plugin/pkg/skplug/proto/skplug.proto

```diff
@@ -60,9 +60,9 @@ message VerticalRecommendationResponse{
 
 message RecommendedPodResources{
   string pod_name = 1;
-  int32 lower_bound = 2;
-  int32 upper_bound = 3;
-  int32 target = 4;
+  int64 lower_bound = 2;
+  int64 upper_bound = 3;
+  int64 target = 4;
```

Comment:
Why `int64`?

Reply:
Because VPA sends int64. As we declared int32 in proto, we end up with type mismatching.

Reply:
The rest of Skenario (and most of Kubernetes) uses int32.  Is there any chance we will overflow?  If not, then I would prefer to cast downward.

Reply:
I think then we can stick to int32 type. I don't think we'll end up with overflowing.